Documentation/operations/upgrade.rst

```diff
@@ -790,14 +802,60 @@ Once the migration is complete, confirm the endpoint identities match by listing
       $ kubectl exec -n kube-system cilium-1234 -- cilium-dbg endpoint list # existing etcd-backed endpoints
 
 Clearing CRD identities
-~~~~~~~~~~~~~~~~~~~~~~~
+$$$$$$$$$$$$$$$$$$$$$$$
 
-If a migration has gone wrong, it possible to start with a clean slate. Ensure that no cilium instances are running with identity-allocation-mode crd and execute:
+If a migration has gone wrong, it possible to start with a clean slate. Ensure that no Cilium instances are running with ``--identity-allocation-mode=crd`` and execute:
 
 .. code-block:: shell-session
 
       $ kubectl delete ciliumid --all
 
+.. _double_write_migration:
+
+Migration with the "Double Write" identity allocation mode
+##########################################################
```

Comment:
I'd suggest adding a note that this is a beta feature, and users may need extra care. This can be done with:

```
.. include:: ../beta.rst
```